testing/python_lib/test_failscale.py

```diff
@@ -26,7 +27,7 @@ def __init__(self, *args, **kwargs):
             yaml.dump(config, config_file)
 
         self.stack_options.update({
-            'devices': self.devices,
+            'devices': 1,
```

Comment:
This doesn't seem right... how is self.devices manifest?  And if the answer is "elsewhere" -- why is this one and not zero?  I'd expect all the devices to be created in one place.

Reply:
Changed to remove devices option in test_scale itself.